nekoyume/Assets/_Scripts/UI/Tutorial/GuideDialog.cs

```diff
@@ -7,6 +6,8 @@
 using Nekoyume.L10n;
 using UnityEngine;
 using RedBlueGames.Tools.TextTyper;
+using UniRx;
```

Comment:
You can using `System` here and using `UniRx` inside of namespace `Nekoyume.UI`.